src/components/EmailEditor/store/emailEditorConfig.js

```diff
@@ -118,6 +118,45 @@ const getDefaultState = () => {
 					canRemoveAll: true,
 				},
 			},
+			components: {
+				image: {
+					add: true,
+					delete: true,
+					save: true,
+					edit: true,
+					reset: true,
+					unlink: true,
+				},
+				text: {
+					add: true,
+					delete: true,
+					save: true,
+					edit: true,
+					reset: true,
+					unlink: true,
+				},
+				video: {
+					add: true,
+					delete: true,
+					save: true,
+					reset: true,
+					ulink: true,
+				},
+				button: {
+					add: true,
+					delete: true,
+					save: true,
+					reset: true,
+					ulink: true,
+				},
+				divider: {
+					add: true,
+					delete: true,
+					save: true,
+					reset: true,
+					ulink: true,
```

Comment:
The `video`, `button` and `divider` elements: 
- Have a small typo with the `unlink` prop
- Are missing the `edit` prop (so it shows `undefined` in the settings right tab)